examples/demo/public/environment.json

```diff
@@ -1,3 +1,4 @@
 {
-  "apiUrl": "https://api.mlab.com/api/1/databases/toolkitaxa/collections/{path}"
+  "apiUrl": "https://api.mlab.com/api/1/databases/toolkitaxa/collections/{path}",
+  "baseUrl" : "build/"
```

Comment:
have you think to update the release srcipts?

Reply:
i was going to make another pull request for this